src/pages/gateway/migration.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Migration
+description: This page describes breaking changes and how to migrate from one version of API Mesh for Adobe Developer App Builder to another.
+---
+
+# Migration
+
+This page lists breaking changes between versions of API Mesh for Adobe Developer App Builder. For a list of new features and resolved issues, refer to the [release notes](release-notes.md).
+
+To update to the latest version, run the following command:
+
+```bash
+npm install -g @adobe/aio-cli-plugin-api-mesh
+```
+
+See our [NPM page](https://www.npmjs.com/package/@adobe/aio-cli-plugin-api-mesh) or [Getting Started](getting-started.md#configure-your-environment) for more information.
+
+## Migrating from Beta to GA
+
+If you participated in the beta for API Mesh, you may need to account for the following breaking changes when updating to the GA release.
+
+### Resolver Composition transform deprecated
+
+The Resolver Composition transform (`composeResolvers`) is no longer supported.
+
+### Hooks transform temporarily disabled
+
+The new [Hooks transform](hooks.md) will be temporarily disabled as part of this release. We will enable this transform in a future release.
+
+### Mesh create and update time
+
+While most [create](command-reference.md#aio-api-meshcreate) and [update](command-reference.md#aio-api-meshupdate) commands will happen very quickly, it can take up to five minutes to fully provision and propagate your mesh.
+
+To address this, we have added the [`aio api-mesh:status` command](command-reference.md#aio-api-meshstatus) which allows you to view where your mesh is in the provisioning process. Additionally, this command will indicate any errors associated with creating your mesh.
```

Comment:
to check the status of your mesh while building if it wasn't instantaneous - something like that